torchaudio/transforms/_transforms.py

```diff
@@ -722,7 +722,7 @@ def __init__(
     def forward(self, waveform: Tensor) -> Tensor:
         r"""
         Args:
-            waveform (Tensor): Tensor of audio of dimension (..., time).
+            waveform (Tensor): Tensor of the audio signal of dimension (..., time).
```

Comment:
Nevermind this change, it seems that I got confused and edited the docstring of the class that was right above the one I created.